src/Faker/Provider/ru_RU/Company.php

```diff
@@ -106,14 +125,44 @@ public static function inn($area_code = "")
         }
         $area_code = str_pad($area_code, 2, '0', STR_PAD_LEFT);
         $inn_base =  $area_code . static::numerify('#######');
-        return $inn_base . \Faker\Calculator\Inn::checksum($inn_base);
+        return $inn_base . self::inn10Checksum($inn_base);
     }
 
     public static function kpp($inn = "")
     {
         if ($inn == "" || strlen($inn) < 4) {
-            $inn = static::inn();
+            $inn = self::inn10();
         }
         return substr($inn, 0, 4) . "01001";
```

Comment:
```suggestion
        return substr($inn, 0, 4) . '01001';
```

Reply:
How about we enable [`single_quote`](https://docs.styleci.io/fixers#single_quote) in StyleCI for these?

Reply:
Couldn't agree more.